docs/v1/use-cases.md

```diff
@@ -3,6 +3,13 @@
 * TODO: scrub doc post glossary
 * TODO: add CF SB API as the lingua franca for blackbox services
 
+## Personas
+
+* User
+* Developer
+* Broker Operator
+* Service Provider
```

Comment:
I wouldn't have "User" as a persona.  What makes user different than developer, that set of things is the persona you are missing.